plugins/GSdx/GLLoader.cpp

```diff
@@ -335,12 +335,12 @@ namespace GLLoader {
 		fprintf(stdout, "OpenGL information. GPU: %s. Vendor: %s. Driver: %s\n", glGetString(GL_RENDERER), vendor, &s[v]);
 
 		// Name changed but driver is still bad!
-		if (strstr(vendor, "ATI") || strstr(vendor, "Advanced Micro Devices"))
+		if (strstr(vendor, "ATI") || strstr(vendor, "Advanced Micro Devices") || strstr(vendor, "ATI Technologies Inc."))
```

Comment:
Done however I won't push it yet , I want to make some other changes.

I noticed the gs version is 1.1.0 shown in the emulator , however checking the dll file version and product version it's 1.0.1.9  
Should this be updated ?